tools/autotools/LogicalImagerPathRule.cpp

```diff
@@ -88,6 +88,11 @@ bool LogicalImagerPathRule::matches(TSK_FS_FILE * /*fs_file*/, const char *path)
     }
 
     for (auto it = std::begin(m_paths); it != std::end(m_paths); ++it) {
+        if (it->find(lowerCaseUserFolder) != std::string::npos) {
+            if (matchUserFolder(*it, lowercasePath)) {
```

Comment:
This isn't compiling - it looks like the lowercasePath part is no longer necessary?